WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -214,6 +220,14 @@ public IEnumerable<HdPubKey> GetKeys(KeyState? keyState = null, bool? isInternal
 			}
 		}
 
+		public IEnumerable<byte[]> GetPubKeyScriptBytes()
+		{
+			lock (HdPubKeysLock)
```

Comment:
@lontivero Do we need this lock here? I cannot really decide, so I leave it.

Reply:
Yes, in the meantime I realized it needs its own lock, since we are not using concurrent collections for performance gain. Fixed.